pages/k8s/1.18/components.md

```diff
@@ -329,34 +331,21 @@ These charms are frequently used with Charmed Kubernetes.
 These are the container images used by this release:
 
 <!-- GENERATED CONTAINER IMAGES -->
--   addon-resizer-amd64:1.8.5
--   addon-resizer-amd64:1.8.5
--   addon-resizer-arm64:1.8.5
--   addon-resizer-arm64:1.8.5
--   addon-resizer-s390x:1.8.5
--   addon-resizer-s390x:1.8.5
+-   cdk/addon-resizer-amd64:1.8.5  
```

Comment:
@kwmonroe Thanks for taking the time to go through this.

i don't have anything specifically against any of the prefixes, I think they just got sedded out when I tried to munge big blobs of text into a sorted list. I see that some of those were overreaching.

As for arches being missing, this is entirely because they weren't in the particular blob I grabbed.

How about we change the Jenkins job to emit a nice list instead?